CHANGELOG.md

```diff
@@ -55,7 +55,12 @@ The format is based on [Keep a Changelog].
   as `IBMQAccountError`). Also, the exception class `IBMQApiUrlError` 
   has been replaced by `IBMQAccountCredentialsInvalidUrl` and 
   `IBMQAccountCredentialsInvalidToken`. (\#480)
-  
+- JSON schema validation is no longer run by default on qobj objects passed
+  to `IBMQBackend.run()`. This siginficantly speeds up the execution of the
+  `run` method and the API server will return 400 if the qobj is invalid. If
+  you would like to still run the validation you can set the `validate_qobj`
+  kwarg to `True` which will enable the JSON schema validation. (\#554)
```

Comment:
This should go under `UNRELEASED`. This section is for already released `0.4.6`.

Reply:
Done in: https://github.com/Qiskit/qiskit-ibmq-provider/pull/554/commits/492edd0405dee2025f8e56af4b325b101afacb26